### PR TITLE
Brand Page

### PR DESCRIPTION
**Pull request by Prid13**
Access at: [wgtwo-dev.netlify.app **/brand**](https://wgtwo-dev.netlify.app/brand)

- Complete replica of https://menu-accept-258355.framer.app/
- NOT yet mobile-friendly
- Would need slight tweaking before migrating to separate subdomain (outside of Docusaurus/React)

Also, I noticed that one color is missing from the framer page: Aurora Purple Lighten 3 and 4 are identitcal!

### CODE REVIEW

File: website/src/pages/brand/index.tsx [hunk 1]
```diff
@@ -0,0 +1,881 @@
...
+        </ul>
+      </div>
+    </section>
+
+    <section className={`${styles.sectionBlue} ${styles.gap40}`} id="downloads">
+      <h1>The good stuff!</h1>
+
+      <h3 className={`${styles.downloadLink} ${styles.fw_300}`}>
+        <a href="#">
+          Download all WG2 logos
```
Comment (jonny-wg2) [inline website/src/pages/brand/index.tsx:822]: Can you make the download link https://drive.google.com/drive/folders/1gm13n5gH7qRm2jDe9bSRBydNc9xMGBjv

**Review by jonny-wg2 — CHANGES_REQUESTED** [review]

**Review by jonny-wg2 — CHANGES_REQUESTED** [review]
I used this page over the week and was super helpful! One nice feature would be adding a download link to download inter / the font. https://fonts.google.com/specimen/Inter. I had the usecase where I was doing something in photoshop and didn't have the link downloaded locally.

Once we make it mobile friendly let's give it a merge!

**Comment by torotimes** [comment]
Hi @jonny-wg2 

Good suggestion. 

@Prid13 wanna have a look at that? 

Tor

**Comment by Prid13** [comment]
Done :)

Screenshot_7381

Another variant:

Screenshot_7376

**Comment by Prid13** [comment]
I've also made the page a bit mobile friendly, _except_ for the navbar. Don't know how to avoid using a hamburger menu 😅

**Review by torotimes — APPROVED** [review]

**Comment by torotimes** [comment]
@jonny-wg2 are you ok to merge this one?

**Review by jonny-wg2 — APPROVED** [review]
🤟🏻

**Comment by jonny-wg2** [comment]
Looks great! Just merged

**Pull request merged**